pom.xml

```diff
@@ -670,10 +671,14 @@
                                     <classPatterns>
                                         <classPattern>uk.ac.cam.cl.dtg.isaac.dto.**DTO</classPattern>
                                     </classPatterns>
+                                    <customTypeMappings>
```

Comment:
Given that these mappings are the same for both executions, you should be able to remove them from here (and the other execution), and place this in the general `configuration` section below. I think that configuration is shared by all the different executions.

Another important thing is that there is a ticket to replace the usages of the old Date/Time library (`java.util`) with the new one (`java.time`). I'll update that ticket so that we don't forget to change this as well.